src/packages/apollo/src/metadata-service/resolver.ts

```diff
@@ -74,10 +74,8 @@ export const getAdminUiMetadataResolver = (hooks?: AdminMetadata['hooks']) => {
 							type: entityName,
 						};
 						const relatedObject = objectTypeData[entityName];
-						if (field.typeOptions.array) {
-							if (!relatedObject) {
-								throw new Error(`Unknown entityName ${entityName}`);
```

Comment:
I think it is reasonable not to throw here as it is legit to have primitives in an array.